src/Fixer/PhpUnit/PhpUnitDedicateAssertInternalTypeFixer.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\PhpUnit;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\Fixer\ConfigurationDefinitionFixerInterface;
+use PhpCsFixer\FixerConfiguration\FixerConfigurationResolver;
+use PhpCsFixer\FixerConfiguration\FixerOptionBuilder;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Indicator\PhpUnitTestCaseIndicator;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+use PhpCsFixer\Tokenizer\TokensAnalyzer;
+
+/**
+ * @author Filippo Tessarotto <zoeslam@gmail.com>
+ */
+final class PhpUnitDedicateAssertInternalTypeFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface
```

Comment:
I see `ConfigurationDefinitionFixerInterface` is deprecated, but I don't find the correct alternative to implement the `createConfigurationDefinition` method: help is welcome (It's a lot of time I can't follow PHP-CS-Fixer internal evolutions :cry:)

Reply:
I think you are looking you are looking for `PhpCsFixer\Fixer\ConfigurableFixerInterface`, if you extend the `AbstractFixer` and implement this interface you can keep the `createConfigurationDefinition` the same